packages/contracts/hardhat.config.ts

```diff
@@ -60,22 +60,30 @@ const config: HardhatUserConfig = {
     compilers: [
       {
         version: '0.7.6',
+        settings: {
+          optimizer: { enabled: true, runs: 10_000 },
+          metadata: {
+            bytecodeHash: 'none',
+          },
+          outputSelection: {
+            '*': {
+              '*': ['storageLayout'],
+            },
+          },
+        },
       },
       {
         version: '0.5.17', // Required for WETH9
-      },
-    ],
-    settings: {
-      optimizer: { enabled: true, runs: 200 },
-      metadata: {
-        bytecodeHash: 'none',
```

Comment:
We want the `.metadata.bytecodeHash` for WETH9?

Reply:
This setting isn't supported by `solc@0.5.17`. Included that field causes a compiler error. 
Is this likely to cause an issue in production?

Reply:
Will not cause an issue in production because we will not modify the content of this contract